github/event_types.go

```diff
@@ -190,6 +190,38 @@ type IssueCommentEvent struct {
 	Sender  *User       `json:"sender,omitempty"`
 }
 
+// Triggered when a repository's label is created, edited, or deleted.
+// The Webhook event name is "label"
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#labelevent
+type LabelEvent struct {
+	//Action is the action that was performed. Possible values are:
```

Comment:
There should be a space here between `//` and `Action`.

See https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes.